deepcell/applications/application.py

```diff
@@ -300,10 +306,10 @@ def _predict_segmentation(self,
         output_tiles = self.model.predict(tiles, batch_size=batch_size)
 
         # Untile images
-        output_images = self._untile_output(output_tiles, tiles_info)
+        self.output_images = self._untile_output(output_tiles, tiles_info)
 
         # Postprocess predictions to create label image
-        label_image = self._postprocess(output_images, **postprocess_kwargs)
+        label_image = self._postprocess(self.output_images, **postprocess_kwargs)
```

Comment:
I am not a big fan of creating attributes outside of `__init__`. Is this required?